licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+# Go to cc-licenses-data and do something
+GO_TO_TRANSLATIONS_REPO = f"cd {TRANSLATION_REPOSITORY_DIRECTORY} && "
+PULL_BRANCHES_DOWN_FROM_DEVELOP = "git checkout develop && git pull"
+PULL_DOWN_TRANSLATION_BRANCHES = (
+    GO_TO_TRANSLATIONS_REPO + PULL_BRANCHES_DOWN_FROM_DEVELOP
+)
+
+
+def run_django_distill():
+    """Outputs the build directory
+    """
+    my_env = os.environ.copy()
+    my_env["DJANGO_SETTINGS_MODULE"] = "cc_licenses.settings.publish"
+    cmd = "python manage.py distill-local --quiet --force"
+    return subprocess.run(
+        cmd, shell=True, check=True, text=True, input="YES", env=my_env,
+    )
```

Comment:
Actually, in this case we don't need to use `subprocess.run` to call another Django management command from within Django. We can just use `call_command` https://docs.djangoproject.com/en/3.0/ref/django-admin/#django.core.management.call_command

Reply:
https://github.com/meeb/django-distill/blob/167c31715cd93e294515fa7edb592adedbed395d/django_distill/management/commands/distill-local.py#L57-L77

Reply:
Is there a way to provide input with `call_command`? If a build dir doesn't exist at the target location, we must provide an input to create the build dir. In the source code for `django-distill` the author doesn't make answering `YES` to all input accessible. `--force` only works to overwrite an existing build dir.